docs/developers/wordle-keeper.md

```diff
@@ -0,0 +1,247 @@
+    }
+    currentCount, err := strconv.Atoi(guess.Count)
+    if err != nil {
+      panic(err)
+    }
+    count = currentCount
+  } else {
+    // Initialize Count Value If No Entry Exists for this Submitter for Today's Wordle
+    count = 0
+  }
+  // Increment Guess Count
+  count += 1
+  var newGuess = types.Guess{
+    Index:     currentTimeGuesserHashString,
+    Submitter: msg.Creator,
+    Word:      submittedSolutionHashString,
+    Count:     strconv.Itoa(count),
+  }
+  // Remove Current Guess Entry to be Updated With New Entry
+  k.RemoveGuess(ctx, currentTimeGuesserHashString)
+  // Add New Guess Entry
+  k.SetGuess(ctx, newGuess)
+  // Setup Reward 
+  reward := sdk.Coins{sdk.NewInt64Coin("WORDLE", 100)}
+  if !(wordle.Word == submittedSolutionHashString) {
+    return &types.MsgSubmitGuessResponse{Title: "Wrong Answer", Body: "Your Guess Was Wrong. Try Again"}, nil
+  } else {
+    // If Submitter Guesses Correctly
+    guesserAddress, _ := sdk.AccAddressFromBech32(msg.Creator)
+    moduleAcct := sdk.AccAddress(crypto.AddressHash([]byte(types.ModuleName)))
+    // Send Reward
+    k.bankKeeper.SendCoins(ctx, guesserAddress, moduleAcct, reward) 
+    return &types.MsgSubmitGuessResponse{Title: "Correct", Body: "You Guessed The Wordle Correctly!"}, nil
+  }
+}
+```
+
+In the above code, we are doing the following things:
+
+* Here, we are doing initial checks again on the word to ensure
+  it’s 5 characters and only alphabet characters are used, which
+  can be refactored in the future or checked within the CLI commands.
+* We then get the Wordle of the Day by getting the hash string of
+  the current day.
+* Next we create a hash string of current day and the Submitter.
+  This allows us to create a Guess type with an index that uses the
+  current day and the address of the submitter. This helps us when we
+  face a new day and an address wants to guess the new wordle of the day.
+  The index setup ensures they can continue guessing a new wordle
+  every day up to the max of 6 tries per day.
+* We then check if that Guess type for the Submitter for today’s
+  wordle did reach 6 counts. If it hasn’t, we increment the count.
+  We then check if the guess is correct. We store the Guess type with
+  the updated count to the state.
+
+## Protobuff File
+
+  A few files need to be modified for this to work.
+
+The first is `proto/wordle/tx.proto`.
+
+Inside this file, fill in the empty `MsgSubmitGuessResponse`
+with the following code:
+
+```go
+message MsgSubmitGuessResponse {
+  string title = 1;
+  string body = 2;
+}
+```
+
+Next file is `x/wordle/types/expected_keepers.go`
+
+Here, we need to add the SendCoins method to the BankKeeper
+interface in order to allow sending the reward to the right guesser.
+
+```go
+type BankKeeper interface {
+  SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) error
```

Comment:
There is a `SpendableCoins` method already there, do we remove this or keep both?

Reply:
Yeah we replace it with `SendCoins`